spec/support/fixtures/applicants.json

```diff
@@ -7,6 +7,10 @@
     {
       "id": "1030303-123123-123145",
       "first_name": "Jane"
+    },
```

Comment:
Why did we need to add another applicant for this? Couldn't we have used an existing one, as we had two anyway? (e.g. someone who has a live video, and someone who doesn't)

Reply:
I wasn't sure what was the correct way to go, I'll use one of the existing ones :)